docs/src/main/asciidoc/security-getting-started.adoc

```diff
@@ -3,70 +3,70 @@ This guide is maintained in the main Quarkus repository
 and pull requests should be submitted there:
 https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc
 ////
-= Getting Started with Quarkus Security
+= Secure a Quarkus application with basic authentication and JPA
```

Comment:
Hello @sberyozkin, @ebullient and @gsmet,

I completed an initial review and applied edits based on some of the style priorities that we discussed as being potential areas to focus on for the Quarkus developer docs. 

Apologies there's a lot in this PR. For future iterations, I'll submit smaller chunks as per the process discussion of late.

Many thanks :-) 
Michelle